### DISCUSSION TITLE
Combo counter not resetting properly on sliderend miss

### DISCUSSION THREAD

**Discussion by vladyslavpron** [ppy/osu#20277] [Q&A]
### Type

Cosmetic

### Bug description

In-game combo counter not resetting on sliderend miss, but in resulting screen its counted properly. Values are different, so this can be considered as bug I think? 
I may attach replay on which it is clearly visible, if needed.
A few days or maybe more passed since I encountered this issue, so my logs might not be useful
 

### Version

2022.910.0

### Logs

[performance-audio.log](https://github.com/ppy/osu/files/9547116/performance-audio.log)
[performance-draw.log](https://github.com/ppy/osu/files/9547117/performance-draw.log)
[performance-input.log](https://github.com/ppy/osu/files/9547118/performance-input.log)
[performance-update.log](https://github.com/ppy/osu/files/9547119/performance-update.log)
[runtime.log](https://github.com/ppy/osu/files/9547120/runtime.log)
[updater.log](https://github.com/ppy/osu/files/9547121/updater.log)
[database.log](https://github.com/ppy/osu/files/9547122/database.log)
[legacy-ipc.log](https://github.com/ppy/osu/files/9547123/legacy-ipc.log)
[network.log](https://github.com/ppy/osu/files/9547124/network.log)
[performance.log](https://github.com/ppy/osu/files/9547125/performance.log)

**Comment by smoogipoo** [comment]
Yes, please attach a replay.

**Comment by vladyslavpron** [comment]
Can't attach replay as file right there, so: 
https://drive.google.com/file/d/1PzdxJuiLDgJd6Ep-Sna1BJQL0mM9Z1uK/view?usp=sharing
As you see from replay, I've missed sliderend on 157 combo, but combo counter continued counting my combo. And in results screen there is 159 max combo, which I've scored later on during this play

**Comment by Flamiii** [comment]
![osu_2022-09-12_18-08-41](https://user-images.githubusercontent.com/74088315/189773783-2d117d00-98c4-491d-b337-12044ca30b79.png)

If this is what you're talking about, this seems like normal behaviour in lazer. Lazer generally has more leniency when it comes to sliders and sliderends don't count towards your combo either. If you want the gameplay to better match stable, try out the Classic mod.

**Reply by vladyslavpron** [reply]
I'm talking about this, but my question is about another thing, explained below in discussion

**Comment by frenzibyte** [comment]
Converted to Q&A until it's clear whether this is an actual issue or just a misunderstanding of lazer's mechanisms.

**Comment by vladyslavpron** [comment]
Okay, even if this sliderend was hit. Then combo in results screen should be more than 159, which I've scored later, so it looks like there are different mechanism for in-game combo counter and combo counter in score statistics, and they are producing different results

**Reply by Flamiii** [reply]
![osu_2022-09-13_01-11-53](https://user-images.githubusercontent.com/74088315/189823479-a6550d24-0876-435e-b62b-cc2b09b6bb15.png)

I see what you mean now, in the replay your max combo should've been 260x but it only shows up as 159x. I think that this is more of an issue with the replay being somewhat bugged and less having to do with actual gameplay. I understand your issue though, as I've had problems with replays of mine counting misses when I didn't miss or vice-versa. That being said, I'm not entirely sure if there's really anything that can be done at the moment since it doesn't seem like there's an easy fix to this issue.

**Reply by vladyslavpron** [reply]
It was in game as well, but yea, it is not critical so can be ignored I guess, thanks
